docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -46,11 +54,16 @@ Step 1: Prerequisites
 
 * You will need the public keys of all the existing BigchainDB nodes.
 
+* Client Certificate for the new BigchainDB Server to identify itself to MongoDB Primary instance.
+
 * A new Kubernetes cluster setup with kubectl configured to access it.
 
 * Some familiarity with deploying a BigchainDB node on Kubernetes.
   See our :doc:`other docs about that <node-on-kubernetes>`.
 
+* *If* you want to deploy monitoring and backup agents as well, you will need a certificate for MongoDB monitoring
```

Comment:
We can definitely introduce multiple instances for monitoring and backup to improve HA/failover, but there is one only active Monitoring and Backup agent in a MMS group and multiple instances work in a Active/Standby mode. But I will remove the optional tag from these.